src/app/components/past-events/past-events.component.html

```diff
@@ -0,0 +1,25 @@
+
+    <div>
+        <div class="center">
+            <div class="center">
+                <p class="semiBold">Events</p>
+            </div>
+
+            <div class="center">
+                <h1>Past Events</h1>
+            </div>
+
+            <div class="center">
+                <p class="medNorm">What we've been up to</p>
+            </div>
+        </div>
+
+        <div class="eventsLayout">
+            <img src="../../assets/images/past-events.png">
+
+            <div class="center rsvpLayout">
+                <button class="button btn">RSVP</button>
```

Comment:
Curious why the `button` and `btn` classes are created separately when they are used just once on the same element, but if it works it works